OpenOversight/app/main/views.py

```diff
@@ -46,8 +45,8 @@ def get_tagger_gallery(page=1):
         return redirect(url_for('main.label_data'), code=307)
 
 
-@main.route('/gallery/<int:page>', methods=['POST'])
-@main.route('/gallery', methods=['POST'])
+@main.route('/gallery/<int:page>', methods=['GET', 'POST'])
+@main.route('/gallery', methods=['GET','POST'])
```

Comment:
what is the actual effect of refreshing the page the visitor ends up on (the one that's been loaded as a GET)? Do the params come along such that folks can still use stuff, or is it an error?